android-components/components/feature/fxsuggest/fxsuggest.fml.yaml

```diff
@@ -21,6 +21,7 @@ features:
         type: Map<SuggestionType, Boolean>
         default: {
           "amp": false,
+          "ampMobile": false,
```

Comment:
Is it possible for these to both be true?

Reply:
No, I don't believe both can be true at the same time